vaccine_feed_ingest/runners/wa/prepmod/normalize.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+
+import calendar
+import datetime
+import json
+import pathlib
+import re
+import sys
+from typing import List
+
+from vaccine_feed_ingest.schema import schema  # noqa: E402
+
+
+def _get_source(site: dict, timestamp: str) -> schema.Source:
+    return schema.Source(
+        source="wa_prepmod",
+        id=site["name"],
+        fetched_from_uri="https://prepmod.doh.wa.gov/clinic/search",
+        fetched_at=timestamp,
+        data=site,
+    )
+
+
+def _get_inventory(site: dict) -> List[schema.Vaccine]:
```

Comment:
I wonder if we could offer something like this as a shared helper which folks could use in several normalizers

Reply:
yeah, agree.